index.html

```diff
@@ -0,0 +1,602 @@
+<!DOCTYPE html>
+<html lang = "en">
+<head>
+<meta charset="UTF-8">
+<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.0/css/all.css" integrity="sha384-lZN37f5QGtY3VHgisS14W3ExzMWZxybE1SJSEsQp9S+oqd12jhcu+A56Ebc1zFSJ" crossorigin="anonymous"><link rel="stylesheet" type = "text/css"  href="style.css" >
+<title>Space Ripples Reveal Big Bang’s Smoking Gun</title>
+
+</head>
+
+<body>
+
+<div class='header'>
+
+    
+    <button class = "buttons" type="submit" value="LOG IN">LOG IN</button>
+
+    <button class = "buttons login" type="submit" value="SUBSCRIBE">SUBSCRIBE NOW</button>
+
+    <img alt="NYT-logo" class="nyt-logo" src = "images/nyt-logo.png">
+    <i class="fas fa-bars"></i>     
+    <i class="fab fa-sistrix"></i>
+    <a class="header-text" href='#'> SPACE & COSMOS</a>
+    
+    
+
+
+
+</div>
+
+
+<div class='img-div'>
+        <h1 class="title">Space Ripples Reveal 
+                <br>Big Bang’s Smoking Gun</h1>
+
+        <img alt="Cosmos image" class="main-image" src = "images/cosmos.jpg">
+
+
+    <p class="text-image">Alan Guth was one of the first physicists to hypothesize the existence of inflation, which explains how the 
+        <br> universe expanded so uniformly and so quickly in the instant after the Big Bang 13.8 billion years ago.</p>
+    <p class="sub-text-image">Rick Friedman for The New York Times    </p>
+
+
+</div>
+
+
+<div class="row">
+    <div class="column left">   </div>
+
+
+
+<div class="column center">
+
+
+
+
+    <h3  class='author-name'>By<a href='#'> Dennis Overbye</a></h3>
+    <h3  class='date-h'> March 17, 2014
+        <i class="fab fa-facebook-f"></i> 
+        <i class="fab fa-twitter"></i> 
+        <i class="fas fa-envelope"></i> 
+        <i class="fas fa-share"></i> 
+        <i class="far fa-bookmark"></i> 
+        <i class="far fa-comment-alt"></i> </h3>
+    <p>
```

Comment:
you are using only one p tag in here but i think it's better to use p tag for every paragraph.